amida-tech/indaba-client
# Feature/inba 288 agree disagree

I am not a proud man.

#### What does this PR do?
Allows stages "Review and Comment" and "Review and Edit" to create commentary against answered surveys. This requires specifications from Kate about proper handling of things, but it's functional. 

#### Related JIRA tickets:

#### How should this be manually tested?
Create everything (project, survey, users...) Create two stages, with a "Review and Comment" or a "Review and Edit" stage as the latter. Have the first one be "Complete Survey."

Assign tasks. Log in as the user who must complete the survey and do so. Hit the "Complete The Task" button. Log out. 

Login as the next user. Go to the next stage's task. Ensure that the answers from the prior stage appear.

On the review pane, choose to agree or disagree with a question/answer. If you choose disagree but do not provide a comment, the comment will not be added or appear when you hit "Submit Review." The review should appear in a window above the Review Pane radio buttons. 
#### Background/Context

#### Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/4026454/34195135-154be8cc-e52a-11e7-8e81-25efacef96d4.png)
